### DISCUSSION TITLE
Can the "PnP Modern Search v4" solution be deployed at the same time at the site collection app catalog and the tenant app catalog without causing issue

### DISCUSSION THREAD

**Discussion by Arioule** [SharePoint/sp-dev-docs#9659] [Q&A]
Hi,

I have deployed the PnP 'Modern Search' solution in a site collection app catalog and we would like know if the deployment of this solution to the tenant app catalog can cause troubles for the site collection where the solution is already there ?
If yes, what can I do to avoid it ?
Thanks a lot for your help,

**Comment by erobillard** [comment]
In fact recent changes - i.e., deprecation of the SPO App Store as of July 1 - seem to force deploying PnP Modern Search to All sites rather than site-by-site (as is more typical with this solution). Existing web parts (i.e. added before the change) still appear on Pages where they've been added and can be configured, but the WPs are no longer visible in the Add Web Part UI to add to a page. The exception seems to be that when the App is deployed to All Sites, the app's web parts are then available, as expected, to add to any given site. 

Note that I've seen this behaviour in two US East tenants, though haven't had it confirmed anywhere else or by anyone else. If there is another potential cause I'd be glad to follow-up on it. 

Have a great day,
-Eli.
